src/v/pandaproxy/schema_registry/storage.h

```diff
@@ -36,7 +36,8 @@
 namespace pandaproxy::schema_registry {
 
 using topic_key_magic = named_type<int32_t, struct topic_key_magic_tag>;
-enum class topic_key_type { noop = 0, schema, config, delete_subject };
+enum class topic_key_type { noop = 0, schema, config, mode, delete_subject };
```

Comment:
> It's safe to change the enum value of `delete_subject` here because we only store this enum as a string, right?

Correct, it's value is never serialised